packages/inference/src/types.ts

```diff
@@ -35,39 +37,7 @@ export interface Options {
 	includeCredentials?: string | boolean;
 }
 
-export type InferenceTask =
-	| "audio-classification"
-	| "audio-to-audio"
-	| "automatic-speech-recognition"
-	| "conversational"
-	| "depth-estimation"
-	| "document-question-answering"
-	| "feature-extraction"
-	| "fill-mask"
-	| "image-classification"
-	| "image-segmentation"
-	| "image-to-image"
-	| "image-to-text"
-	| "object-detection"
-	| "video-classification"
-	| "question-answering"
-	| "reinforcement-learning"
-	| "sentence-similarity"
-	| "summarization"
-	| "table-question-answering"
-	| "tabular-classification"
-	| "tabular-regression"
-	| "text-classification"
-	| "text-generation"
-	| "text-to-image"
-	| "text-to-speech"
-	| "text-to-video"
-	| "token-classification"
-	| "translation"
-	| "unconditional-image-generation"
-	| "visual-question-answering"
-	| "zero-shot-classification"
-	| "zero-shot-image-classification";
+export type InferenceTask = Exclude<PipelineType, "other">;
```

Comment:
this could be defined in tasks directly as it might be helpful elsewhere too.

You could name it `ConcretePipelineType` or something (or `InferenceTask` is fine too i guess)